tensorflow_addons/custom_ops/image/cc/kernels/euclidean_distance_transform_op.cc

```diff
@@ -85,11 +97,11 @@ typedef Eigen::GpuDevice GPUDevice;
 
 namespace functor {
 
-#define DECLARE_FUNCTOR(TYPE)                                               \
-  template <>                                                               \
-  void EuclideanDistanceTransformFunctor<GPUDevice, TYPE>::operator()(      \
-      const GPUDevice &device, OutputType *output, const InputType &images) \
-      const;                                                                \
+#define DECLARE_FUNCTOR(TYPE)                                            \
+  template <>                                                            \
+  void EuclideanDistanceTransformFunctor<GPUDevice, TYPE>::operator()(   \
+      OpKernelContext *ctx, OutputType *output, const InputType &images) \
+      const;                                                             \
   extern template struct EuclideanDistanceTransformFunctor<GPUDevice, TYPE>
 
 TF_CALL_half(DECLARE_FUNCTOR);
```

Comment:
Why we don't register a uint8 kernel

Reply:
Actually, I don't know, but we can do it now :-)